packages/rum/src/domain/record/utils.ts

```diff
@@ -8,3 +10,11 @@ export function forEach<List extends { [index: number]: any }>(
 ) {
   Array.prototype.forEach.call(list, callback as any)
 }
+
+export function getFrustrationFromAction(frustrations: FrustrationType[]): FrustrationType {
+  return frustrations.some((f) => f === FrustrationType.RAGE_CLICK)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏use `includes` from `@datadog/browser-core` instead of `some()`